malariagen_data/ag3.py

```diff
@@ -61,6 +63,7 @@ class Ag3:
     """
 
     contigs = CONTIGS
+    Region = namedtuple("Region", ["contig", "start", "end"])
```

Comment:
Suggest to move this outside the class definition, so it's defined at the module level.